docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -693,6 +694,10 @@ escaped to allow inclusion of whitespace and other control characters.
 
 This key can be listed multiple times.
 
+### `KubeDownForce=`
+
+Remove all resources including volumes when calling `podman kube down`.
```

Comment:
Missing commas: should be "...all resources, including volumes, when ...".

Reply:
Thanks, fixed